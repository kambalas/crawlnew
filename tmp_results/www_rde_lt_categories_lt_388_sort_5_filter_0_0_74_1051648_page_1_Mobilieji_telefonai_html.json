[
  {
    "name": "iPhone 15 256GB Blue Apple",
    "price": "899.67",
    "error": false
  },
  {
    "name": "iPhone 15 128GB Yellow Apple",
    "price": "752.07",
    "error": false
  },
  {
    "name": "iPhone 15 128GB Black Apple",
    "price": "752.07",
    "error": false
  },
  {
    "name": "iPhone 15 128GB Blue Apple",
    "price": "752.07",
    "error": false
  },
  {
    "name": "iPhone 15 128GB Pink Apple",
    "price": "769.00",
    "error": false
  },
  {
    "name": "iPhone 15 128GB Green Apple",
    "price": "769.00",
    "error": false
  },
  {
    "name": "iPhone 15 256GB Black Apple",
    "price": "1044.06",
    "error": false
  },
  {
    "name": "iPhone 15 256GB Green Apple",
    "price": "874.28",
    "error": false
  },
  {
    "name": "iPhone 15 256GB Yellow Apple",
    "price": "902.00",
    "error": false
  },
  {
    "name": "iPhone 15 256GB Pink Apple",
    "price": "946.00",
    "error": false
  },
  {
    "name": "iPhone 15 512GB Pink Apple",
    "price": "1199.00",
    "error": false
  },
  {
    "name": "iPhone 15 512GB Green Apple",
    "price": "1206.69",
    "error": false
  },
  {
    "name": "iPhone 15 512GB Black Apple",
    "price": "1213.42",
    "error": false
  },
  {
    "name": "iPhone 15 512GB Blue Apple",
    "price": "1296.12",
    "error": false
  },
  {
    "name": "iPhone 15 512GB Yellow Apple",
    "price": "1382.45",
    "error": false
  }
]