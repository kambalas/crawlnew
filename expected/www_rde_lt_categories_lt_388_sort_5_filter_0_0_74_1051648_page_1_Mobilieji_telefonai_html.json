[
  {
    "index": 0,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  },
  {
    "name": "iPhone 15 512GB Green Apple",
    "price": "1206.69",
    "error": false
  },
  {
    "name": "iPhone 15 512GB Black Apple",
    "price": "1213.42",
    "error": false
  },
  {
    "name": "iPhone 15 512GB Yellow",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 13 Pro Max 256GB - Graphite",
    "price": "999",
    "error": false
  },
  {
    "name": "iPhone 13 128GB - Pink",
    "price": "799",
    "error": false
  },
  {
    "name": "Apple iPhone SE (2022) 64GB - White",
    "price": "429",
    "error": false
  },
  {
    "index": 1,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'list' object has no attribute 'usage'"
  },
  {
    "index": 2,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'list' object has no attribute 'usage'"
  },
  {
    "name": "iPhone 13 128GB Midnight",
    "price": "514.96",
    "error": false
  },
  {
    "name": "iPhone 11",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 12",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 13",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 14",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 14 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 14 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone SE 4",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone SE",
    "price": "",
    "error": false
  }
]